internal/rbd/rbd_attach.go

```diff
@@ -388,6 +388,33 @@ func appendKRbdDeviceTypeAndOptions(cmdArgs []string, userOptions string) []stri
 	return cmdArgs
 }
 
+// appendCrushLocationMapOptions parses crushLocationMap to the format
+// "--read_from_replica=localize,--crush_location=key1:value1|key3:value2"
```

Comment:
Actual parsing happens in `getCrushLocationMap`:
```suggestion
// appendCrushLocationMapOptions uses crushLocationMap to the format
// "read_from_replica=localize,crush_location=key1:value1|key2:value2"
```

Reply:
changed it to `converts` now